src/index.html

```diff
@@ -1,17 +1,163 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="groupField">
+        <legend>Personal information:</legend>
+        <label for="surname" class="form-field">
+          Surname:
+          <input 
+              type="text" 
+              name="surname" 
+              autocomplete="off"
+          >
+        </label>
+
+        <label for="name" class="form-field">
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            required 
+            autocomplete="off"
+          >
+        </label>
+      
+        <label for="age" class="form-field">
+          How old are You?
+          <input 
+          type="number" 
+          name="age" 
+          value="12" 
+          min="1" 
+          max="100"
+          >
+        </label>
+      
+        <label for="dateOfBirth" class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            value="mm/dd/yyyy"
+          >
+        </label>
+      
+        <label for="terms">I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+
+      <fieldset class="groupField">
+        <legend>Registration:</legend>
+        <label for="email" class="form-field">
+          E-mail:
+          <input 
+          type="email" 
+          name="email" 
+          placeholder="email@example.com" 
+          required
+          >
+        </label>
+      
+        <label for="password">
+          Password:
+          <input 
+          type="password" 
+          name="password" 
+          required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="groupField">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input type="radio" name="cat">
+            Yes
+          </label>
+          <label>
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
          </label>
          
          <label>
```